### PR TITLE
docs(authors/license): move to root

### PR DESCRIPTION
**Pull request by AriLFrankel**
Adds an AUTHORS file modeled after the one in Node (all contributors).
Adds a LICENSE (MIT)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,8 +18,6 @@
     "type": "git",
     "url": "git+https://github.com/ConciergeAuctions/styled-material-components.git"
   },
-  "author": "Brad Decker <bhdecker84@gmail.com>",
-  "license": "ISC",
```
Comment (brad-decker) [inline package.json]: Can you confirm that npm will pick up the license from the license file for display on the npm page?

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+David Lee <david.lee@merlinlabs.com>
+Natalia Bernard <natalia.bernard@merlinlabs.com>
+Marta Johnson <marta.johnson@merlinlabs.com
+Nathan Schwartz <nathan.schwartz@merlinlabs.com
+Krista Goralczyk <krista.goralczyk@merlinlabs.com
+Daniel St.Clair <daniel.stclair@merlinlabs.com
+Jean Page <jean.page@merlinlabs.com>
+Bennett Staley <bennett.staley@merlinlabs.com>
+Ari Frankel <ari.frankel@merlinlabs.com>
+Pete Givens <pgivens@gmail.com>
```
Comment (brad-decker) [inline AUTHORS]: @petegivens :clap:

File: AUTHORS [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#Authors
+
+Brad Decker <brad.decker@merlinlabs.com>
+David Lee <david.lee@merlinlabs.com>
+Natalia Bernard <natalia.bernard@merlinlabs.com>
+Marta Johnson <marta.johnson@merlinlabs.com
+Nathan Schwartz <nathan.schwartz@merlinlabs.com
+Krista Goralczyk <krista.goralczyk@merlinlabs.com
```
Comment (brad-decker) [inline AUTHORS]: @kristajg -- what email do you want here?

File: package.json (hunk 1 above)
Comment (AriLFrankel) [reply package.json]: Looks like it's needed. I will add it back in 

https://github.com/npm/npm/issues/8795

File: AUTHORS (hunk 2 above)
Comment (petegivens) [reply AUTHORS]: 🕵️

Comment (petegivens) [reply AUTHORS]: 🕵️

File: AUTHORS (hunk 3 above)
Comment (AriLFrankel) [reply AUTHORS]: she said krista.goralczyk@gmail.com. I'll add that in.

File: AUTHORS [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+Brad Decker <brad.decker@merlinlabs.com>
+David Lee <david.lee@merlinlabs.com>
+Natalia Bernard <natalia.bernard@merlinlabs.com>
+Marta Johnson <marta.johnson@merlinlabs.com>
+Nathan Schwartz <nathan.schwartz@merlinlabs.com>
+Krista Goralczyk <krista.goralczyk@gmail.com>
+Daniel St.Clair <daniel.stclair@merlinlabs.com>
+Jean Page <jean.page@merlinlabs.com>
+Bennett Staley <bennett.staley@merlinlabs.com>
+Ari Frankel <ari.frankel@merlinlabs.com>
```
Comment (ashwoodall) [inline AUTHORS]: Ashley Woodall <ashley.woodall@merlinlabs.com> pls 😢

Comment (brad-decker) [reply AUTHORS]: Boo @AriLFrankel

Comment (dekkofilms) [reply AUTHORS]: *cough*

Comment (AriLFrankel) [reply AUTHORS]: yes, boooooo @arilfrankel

but also, @ashwoodall you should put in a pr to add yourself as a contributor to Merlin :)

Comment (ashwoodall) [reply AUTHORS]: Fair enough, will do 😎

**Comment by martahj** [comment]
@brad-decker @petegivens @dekkofilms Did you want to re-review this before it gets merged?

**Pull request merged**